activejdbc/src/main/java/org/javalite/activejdbc/dialects/DefaultDialect.java

```diff
@@ -219,4 +219,36 @@ public String insert(MetaModel metaModel, Map<String, Object> attributes) {
         }
         return query.toString();
     }
+    
+    @Override
+    public String update(MetaModel metaModel, Map<String, Object> attributes) {
+    	  StringBuilder query = new StringBuilder().append("UPDATE ").append(metaModel.getTableName()).append(' ');
+          if (attributes.isEmpty()) {
+              appendEmptyRow(metaModel, query);
+          } else {
+        	     
+        	  query.append("SET ");
+        	     
+        	  Iterator<String> keysetIt = attributes.keySet().iterator();
+              Iterator<Object> valueIt = attributes.values().iterator();
+         
```

Comment:
You better use `attributes.entrySet()` instead of separately using `keySet()` and `values()`.